core/class/luna.class.php

```diff
@@ -339,7 +339,7 @@
     return $return;
   }
 
-  public static function saveWifi($interface = 1, $data) {
+  public static function saveWifi($data, $interface = 1) {
```

Comment:
The saveWifi function has changed the order of its parameters. Make sure to update all calls to this function throughout the codebase to reflect this change. [important]